pkgs/by-name/eu/eudic/package.nix

```
@@ -0,0 +1,87 @@
+{ fetchurl
+, stdenv
+, autoPatchelfHook
+, makeWrapper
+, lib
+, copyDesktopItems
+, libnotify
+, libX11
+, libXScrnSaver
+, libXext
+, libXtst
+, libuuid
+, libsecret
+, xdg-utils
+, xdg-utils-cxx
+, at-spi2-atk
+# additional dependencies autoPatchelfHook discovered
+, gtk3
+, alsa-lib
+, e2fsprogs
+, nss
+, libgpg-error
+, libjack2
+, mesa
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "eudic";
+  version = "13.5.2";
+
+  src = fetchurl {
```

```suggestion
  src = fetchzip {
```

Try it, and remove unpacking